src/chess/piece/Pawn.java

```diff
@@ -1,24 +1,81 @@
 package chess.piece;
 
-import java.util.HashMap;
-import java.util.Map;
+import chess.Position;
+
+import java.util.*;
 
 public class Pawn extends Piece {
   private static final Map<Color, Character> symbols = new HashMap<Color, Character>(){{
     put(Color.WHITE, '♙');
     put(Color.BLACK, '♟');
   }};
 
+  // if this pawn is moved, isFirstMoved turn out true but at next your own turn, it turn out false
+  // this variable is related to "en passant"
+  // ex. n:false -> n + 1:true -> n + 2:false
+  private boolean isFirstMoved;
+
+  // if this pawn is moved, isMoved turn out true and it never change again
+  // when the value is true, this Pawn can move forward two squares.
+  // ex. n:false -> n + 1:true
+  private boolean isMoved;
+
   public Pawn(Color color) {
     super(color, symbols);
+    this.isFirstMoved = false;
+    this.isMoved = false;
   }
 
   @Override
-  public boolean isValidMove(Position newPosition) {
-    if (!super.isValidMove(newPosition)) {
+  public boolean isValidMove(int row, int col, boolean isEnemyPiece) {
+    if (!super.isValidMove(row, col, isEnemyPiece)) {
       return false;
     }
-    // ToDo: implement
-    return true;
+
+    // if arguments are impossible int
+    // row = 1 or 2, col = 0 or 1
```

Comment:
umm, something is not clear for me
I thought `row` stands for the change in X-axis direction and `col` stands for the change in Y-axis direction
If you don't agree with this, I think we should use `x` and `y` or `forward` and `right` or some other options instead